api/jsonapi.go

```diff
@@ -1726,7 +1726,6 @@ func (i *jsonAPIHandler) GETModerators(w http.ResponseWriter, r *http.Request) {
 					j++
 				}
 			}
-			xs = (xs)[:j]
```

Comment:
Why is this not needed?

Reply:
This spot is actually a bug, which I missed initially. I'll fix it instead of just removing the assignment.